src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,34 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+  // Create an array from the object values and create a copy also.
+  let arrFromObj = Object.values(obj);
+  let arrFromObjCopy = arrFromObj.map((x) => x);
+  //   Create a new array with the sub arrays equal to the object value sub arrays
+  let newArr = new Array(arrFromObj.length);
+  let flattenedArr;
+  let outputObject = {};
+
+  for (let i = 0; i < arrFromObj.length; i++) {
+    //   Insert empty strings equal to the number of strings in object value array strings
+    newArr[i] = new Array(arrFromObj[i].length);
+    arrFromObjCopy.shift();
+    flattenedArr = arrFromObjCopy.flat();
```

Comment:
Nice. This is a recent addition to javascript, so, might not be available in older versions.